README.md

```diff
@@ -25,6 +20,30 @@ $ git clone https://github.com/user-cont/colin.git
 $ pip3 install ./colin --user
 ```
 
+## Usage
+
+### Directly from git
+
+```
+$ python3 -m colin.cli.colin -h
```

Comment:
Add before running help these two lines:

```
$ git clone https://github.com/user-cont/colin && cd colin
$ pip-3 install . --user
```

Reply:
Give me a mmnt. Testing it inside container.

Reply:
If you don't need it now, I will take a WFT (work-from-train..;-) and implement the loading from the file tonight.

Reply:
Clarified with this Dockerfile:
```
FROM fedora:27

RUN dnf install -y python3-pip git libattr-devel gcc redhat-rpm-config \
                   python3-devel \
    && dnf clean all

RUN git clone https://github.com/user-cont/colin /files/colin \
    && cd /files/colin \
    && pip-3 install . --user \
    && python3 -m colin.cli.colin -h

```

Reply:
Good suggestion, I updated the readme a bit more.

Reply:
Here, the help output should be updated as well.